src/documentation/tools/RateSpread.jsx

```diff
@@ -1,46 +1,113 @@
 import React from 'react'
 import Product from '../Product.jsx'
+import ExternalLink from '../../common/ExternalLink.jsx'
 
 const links = {
   2017: [],
   2018: [
-    <li key="0"><a target="_blank" rel="noopener noreferrer" href="/tools/rate-spread/requirements">Data Requirements</a></li>,
-    <li key="1"><a target="_blank" rel="noopener noreferrer" href="/tools/rate-spread/methodology">Methodology for Determining Average Prime Offer Rates</a></li>
+    <li key='0'>
+      <a
+        target='_blank'
+        rel='noopener noreferrer'
+        href='/tools/rate-spread/requirements'
+      >
+        Data Requirements
+      </a>
+    </li>,
+    <li key='1'>
+      <a
+        target='_blank'
+        rel='noopener noreferrer'
+        href='/tools/rate-spread/methodology'
+      >
+        Methodology for Determining Average Prime Offer Rates
+      </a>
+    </li>,
   ],
   2019: [
-    <li key="2019-0"><a target="_blank" rel="noopener noreferrer" href="https://ffiec.cfpb.gov/tools/rate-spread/requirements">Data Requirements</a></li>,
-    <li key="2019-1"><a target="_blank" rel="noopener noreferrer" href="https://ffiec.cfpb.gov/tools/rate-spread/methodology">Methodology for Determining Average Prime Offer Rates</a></li>
+    <li key='2019-0'>
+      <ExternalLink
+        url={'https://ffiec.cfpb.gov/tools/rate-spread/requirements'}
+        text='Data Requirements'
+      />
+    </li>,
+    <li key='2019-1'>
+      <ExternalLink
+        url={'https://ffiec.cfpb.gov/tools/rate-spread/methodology'}
```

Comment:
This is not external since it's still on `https://ffiec.cfpb.gov`, same for all 2019+.  Looks like we can copy the pattern from 2018 instead.

Reply:
Has been updated.